### PR TITLE
sys-fs/duperemove: add patch

### PR DESCRIPTION
**Pull request by fturco**
See the following links for details
* https://bugs.gentoo.org/show_bug.cgi?id=607376
* https://github.com/markfasheh/duperemove/pull/169

### CODE REVIEW

File: sys-fs/duperemove/duperemove-0.11_beta4.ebuild [hunk 1]
```diff
@@ -23,6 +23,11 @@ DEPEND="${RDEPEND}"
 
 S=${WORKDIR}/${P/_/.}
 
+src_prepare() {
+	epatch "${FILESDIR}/${P}-sysmacros.patch"
```
Comment (mgorny) [inline sys-fs/duperemove/duperemove-0.11_beta4.ebuild]: `epatch` is deprecated, use `eapply` in EAPI 6.

**Comment by fturco** [comment]
@mgorny: is this OK now? I'm still a git beginner...

**Comment by candrews** [comment]
Instead of using `eapply` and overriding `src_prepare` to do so, since you're using EAPI=6, you can use PATCHES:
`PATCHES=( "${FILESDIR}/${P}-sysmacros.patch" )`
making the ebuild change be only one line :)

See https://devmanual.gentoo.org/ebuild-writing/eapi/

**Comment by fturco** [comment]
Thanks @candrews for the suggestion. The ebuild looks cleaner now!
I forgot to say that beside being a beginner with git I'm also a beginner with ebuilds... Please be patient with me, I'm learning :)

**Comment by mgorny** [comment]
Please see the following pages about git commit standards in Gentoo:
- https://wiki.gentoo.org/wiki/User:MGorny/GLEP:Git
- https://wiki.gentoo.org/wiki/Gentoo_GitHub

**Pull request closed without merging**